tests/queries/0_stateless/02247_written_bytes_quota.sh

```diff
@@ -16,10 +16,13 @@ ${CLICKHOUSE_CLIENT} -q "CREATE ROLE r02247"
 ${CLICKHOUSE_CLIENT} -q "CREATE USER u02247"
 ${CLICKHOUSE_CLIENT} -q "GRANT ALL ON *.* TO r02247"
 ${CLICKHOUSE_CLIENT} -q "GRANT r02247 to u02247"
-${CLICKHOUSE_CLIENT} -q "CREATE QUOTA q02247 FOR INTERVAL 100 YEAR MAX WRITTEN BYTES = 25 TO r02247"
+${CLICKHOUSE_CLIENT} -q "CREATE QUOTA q02247 FOR INTERVAL 100 YEAR MAX WRITTEN BYTES = 30 TO r02247"
```

Comment:
It looks that quota is accounted twice for user now.

Reply:
I'll fix.

Reply:
Ah, ok, I understood. Now the size in bytes in block is used, which is slightly larger than number of bytes in string.

Reply:
Now the value 'qwqw' means 13 bytes will be written no matter if it's `async_insert` or not.
Earlier the quota was used differently - 13 bytes for non-async insert of `qwqw` and 8 bytes for async insert of `qwqw`. Of course both inserts write the same blocks of data, it's just only the quota calculation which changed.